### PR TITLE
Setting up redirects

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by thrau — COMMENTED** [review]
looks awesome! glad this works, and no idea why it didn't work on the website :shrug: 

thanks @lukqw for the solution :100: 

```console
thomas@om ~ % curl -L https://localstack-docs-preview-pr-357.surge.sh/ci/circle-ci
<!doctype html><html lang=en><head><title>/user-guide/ci/circle-ci/</title><link rel=canonical href=/user-guide/ci/circle-ci/><meta name=robots content="noindex"><meta charset=utf-8><meta http-equiv=refresh content="0; url=/user-guide/ci/circle-ci/"></head></html>%  
```

just need the other sections now :-)

**Comment by HarshCasper** [comment]
@thrau @lukqw The redirects are completed. Wrote a dirty script if you are looking to use something to test: https://gist.github.com/HarshCasper/fed44e7c3a61853a13342e671833e642

File: content/en/user-guide/tools/multi-account-setups/index.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ categories: ["LocalStack Pro"]
 tags: ["multi-tenant", "multi-account", "account-id", "namespaces"]
 aliases:
   - /aws/multi-account-setups/
```
Comment (lukqw) [inline content/en/user-guide/tools/multi-account-setups/index.md:7]: do we still need this alias?

**Review by lukqw — APPROVED** [review]
Nice work @HarshCasper! Thanks for adding all of the redirects 🚀

File: content/en/user-guide/tools/multi-account-setups/index.md (hunk 1 above)
Comment (HarshCasper) [reply content/en/user-guide/tools/multi-account-setups/index.md:7]: Possibly yes, since this was part of AWS docs before

**Pull request merged**